_docs/110_faq.md

```diff
@@ -8,17 +8,6 @@ permalink: /docs/faq
 
 ## Usage
 
-### I just cloned my repository and conflicting data was overwritten. Why?
```

Comment:
Why remove this section? It is still relevant.

Reply:
The new version of clone does not stash differences anymore. It merely leaves the files as modified.